### PR TITLE
Updated Fonts in Navbar

### PR DESCRIPTION
**Pull request by codedbyyashwanth**

### CODE REVIEW

**Comment by codedbyyashwanth** [comment]
Updated the Navbar UI

**Comment by codedbyyashwanth** [comment]
Updated the UI

File: components/navbar.js [hunk 1]
```diff
@@ -108,12 +108,17 @@ const DesktopNav = () => {
...
+                        fontFamily={'sora'}
                         fontWeight={500}
+                        transition={'ease 0.4s'}
+                        textTransform={'uppercase'}
                         color={'white'}
                         _hover={{
                             textDecoration: 'none',
-                            fontSize: '2xl',
+                            fontSize: '1xl',
+                            color:"#BB9254"
```
Comment (jamesofoaye) [inline components/navbar.js:121]: use `brand.sand` as the color.
the brand colors can be found in the _app file

File: components/navbar.js [hunk 2]
```diff
@@ -108,12 +108,17 @@ const DesktopNav = () => {
                     <Link
                         p={2}
                         href={navItem.href ?? '#'}
-                        fontSize={'lg'}
+                        fontSize={'sm'}
+                        letterSpacing={1.2}
+                        fontFamily={'sora'}
```
Comment (jamesofoaye) [inline components/navbar.js:113]: Use `Lora` font
Lora is part of the brand font

File: components/navbar.js [hunk 3]
```diff
@@ -145,10 +150,15 @@ const MobileNavItem = ({ label, href }) => {
                 py={2}
                 as={Link}
                 href={href ?? '#'}
-                justify={'space-between'}
+                justify={'space-between'}fontFamily={'sora'}
```
Comment (jamesofoaye) [inline components/navbar.js:153]: Use `Lora` font
Lora is part of the brand font

File: components/navbar.js [hunk 4]
```diff
@@ -145,10 +150,15 @@ const MobileNavItem = ({ label, href }) => {
...
+                fontWeight={500}
+                transition={'ease 0.4s'}
+                textTransform={'uppercase'}
                 align={'center'}
                 _hover={{
                     textDecoration: 'none',
+                    fontSize: '1xl',
+                    color:"#BB9254"
```
Comment (jamesofoaye) [inline components/navbar.js:161]: use `brand.sand` as the color.
the brand colors can be found in the _app file

**Review by jamesofoaye — CHANGES_REQUESTED** [review]

**Pull request closed without merging**